.github/pull_request_template.md

```diff
@@ -0,0 +1,6 @@
+Checklist for reviewer:
+
+- [ ] The description should reference a bug or github issue, if relevant.
+- [ ] There must be a [`CHANGELOG.md`](./CHANGELOG.md) entry for any non-test change.
+- [ ] Any change to the NPM commands must be carefully reviewed to make sure it won't break the Add-ons pipeline.
```

Comment:
It's hard to test this without opening a PR over there :/ which (with our current process) we don't notice til release time. I think this warning is OK for now but maybe that's something we could automate in the future.

Reply:
I think we should probably add a minimal test to make sure we don't break upstream. Let me file an issue for this.

Reply:
https://github.com/mozilla-ion/ion-core-addon/issues/261